### ISSUE TITLE
Research | Enhanced Military Info | Conversation guide

### ISSUE THREAD

**Issue by pamela-drouin** [department-of-veterans-affairs/va.gov-team#74208]
Once the [research plan](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73363) is complete, write the conversation guide.

### Tasks:

- [x] Create initial draft of [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Research/2024-01-military-info-enhancement-mvp/conversation-guide.md)
- [x] Solicit review/feedback from UX, FE, and BE
- [x] Solicit review/feedback from PO and PM

### Acceptance Criteria

- [x] Conversation guide is written
- [x] All feedback has been incorporated 
- [x] Guide is ready for research review/recruitment

**Comment by pamela-drouin** [comment]
@mtcA6 I need to make a separate ticket for what might be a blocker for this ticket -- a ticket for Design/FE/BE to have a plan for acquiring research participant military history ahead of time.

We'll need to get that data in order to format it and get it into a Mural board so we can do lightweight mapping with research participants.

**Comment by pamela-drouin** [comment]
@mtcA6 carry over 3 points 😄

**Comment by pamela-drouin** [comment]
@andaleliz I would love peer feedback on the [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/Research/2024-01-military-info-enhancement-mvp/conversation-guide.md) if you have time/capacity!

**Comment by andaleliz** [comment]
@pamela-drouin wonderful job, as always! I have mostly "take it or leave it" suggestions and a couple of things I called out that would be great to get extra info on. Let me know if you'd like me to clarify anything for you. I'm so thankful you're on the team to lead this research!


### Day of the session

I’m sure you won’t but just want to add a note not to forget to fill these out! I forgot for my current study, and then I even forgot to check them in *both* pilots. Eek.


### Intro
> Today we're going to talk about your military history – specifically your military occupational specialty history – basically, the kinds of jobs you did while you were serving/enlisted.  
> 
> I’d like to learn about how you [think about/perceive/regard/conceptualize] your military history – especially your occupational history, and ways in which you have had to share your military history in order to receive benefits or for other reasons.

I’m probably overthinking this, but I can imagine someone who runs anxious, like myself, hearing “occupational speciality history” and not know what it means, and get caught up in their head about it, and then not listen closely to what follows. It might be worth while to switch the order and do a quick gut check:
“Today we're going to talk about the kinds of jobs you did while you were serving/enlisted.  Are you familiar with the phrase “military occupational speciality” or MOS? [Great, just wanted to be sure / No problem, that means XYZ]. We’re going to specifically focus on that today, and I’d like to learn… ”

### Interview questions 

>  Before we go to the next part, do you recall giving permission to gather your military history ahead of time as part of this research session?

**THIS IS SO SMART.** You may want to add a back-up script here if they say they don’t recall doing that and are not comfortable moving forward.

### Mapping activity - 15 minutes
- What do the numbers in parentheses mean next to each section of the activity? Curious if it means number of minutes or if you’re going to limit it to 5 items under each one.
- This comment could be totally moot depending on the data you’ll pull, but you might want to have some items in your pocket to talk to them about if they say nothing is missing. I know we’ve seen some records that have X but not Y, and Y but not X. I don’t think it’s been totally clear to us why those aren’t there, but it could be interesting to talk them about it while they’re looking at their own data. Or maybe it’s something that comes up during the next portion when you start reviewing the mock up :) 

## Mock-up evaluation – 15 minutes

> In Mural, show the wireframe

You are a genius, I love how you’ve set this whole thing up in one Mural board instead of bouncing between multiple tools.

> Include stickies with questions (gradually unhide them)

Are the stickies going to be the bulleted list that follows in the convo guide?

> How would this help you in applying for a new benefit or submitting a claim?

- Might want to specify “How would *having this on the website/in your profile*”
- If you can elicit information here to help us understand if a copy/paste function is something people expect or would use, that would be helpful.

Overall comment: this may come up organically, but if it doesn’t, it’d be helpful to get feedback on the “show more” functionality and what they’d expect from that. Anything to help us understand the following would be very useful:
- if it would be better to tease out a little bit of the info before the “show more”, or if it’s better to stash it all away
- anything to tell us about expectations for having this on a separate page
  - perhaps a sense of how much data appearing upon the click of that button would be overwhelming
- Difference in expectations around active duty vs inactive duty periods

### Post-Task Interview - 2 minutes
- Often times, the “anything else you’d like me to know” question yields a “No, but…[thing they want you to know or long story]" so you may want to account for this taking more than 2 minutes.
- I forgot to add the magic wand question at the end of my current study, and I regret it! I’m going to add it for my sessions tomorrow. Maybe a good note to end on for yours if you have nothing else? “if you had a magic wand, what would you change about what you reviewed today?”

**Comment by Samara-Strauss** [comment]
### Interview questions - 15 minutes

> What was your last military occupational specialty before you retired from the military

There is a difference between separating from the military and retiring from the military. People who serve < 20 years separate; people who serve 20+ years retire. Those who stay through and qualify for retirement are eligible for additional benefits through DoD and maybe VA (not sure about VA). 

TL;DR -- I would use different language like "when did you leave/separate/etc..."

> What are some examples when you had to share your MOS data

This question isn't totally clear to me -- do you mean share it with VA?

### Mock-up evaluation – 15 minutes

> How would having this on the website (in your profile) help you in applying for a new benefit or submitting a claim?

Can you make this more open-ended to something like "How would you use this information if it was on VA.gov?"

> How would this make that difficult? (i.e. is it missing anything key)

I think this should just be "Is anything missing?"

Great job, Pam!!

**Comment by pamela-drouin** [comment]
@mtcA6 @Samara-Strauss @andaleliz -- I have incorporated Samara's feedback! I'd like to submit this for Shane's approval today if possible!

I'm looking at the calendar in terms of starting the study -- how about doing research sessions Monday, February 19 - Wednesday, February 21st? If you all are on board with that, I can update the research plan with those dates and submit for approval.

**Comment by andaleliz** [comment]
@pamela-drouin that sounds good to me! I will be off on 2/19, and traveling back home on 2/20 until about 3pm. That means I won't be able to observe any sessions before that or help with notes. I'm personally ok with that if you are - I completely trust that you're going to elicit what we need from this study.

**Comment by mtcA6** [comment]
@pamela-drouin I'm okay w/that, only qualm, I think the 19th is presidents day so it's a federal holiday but that week is fine w/me

**Comment by pamela-drouin** [comment]
@mtcA6 yeah, I realized that. I'd still like to try and do sessions that day.

@andaleliz thanks for the heads-up!
